include/micm/configure/solver_config.hpp

```diff
@@ -119,7 +119,7 @@ namespace micm
 
   class JsonReaderPolicy
   {
-    using json = nlohmann::json;
+    using json = YAML::Node;
```

Comment:
```suggestion
    using object = YAML::Node;
```

Maybe we should rename this so that it no longer seems specific to json?